scripts/validate-notebooks.ps1

```diff
@@ -34,7 +34,14 @@ Param(
 )
 
 
-& "$PSScriptRoot/install-iqsharp.ps1"
+# Detect if we're running on Azure Pipelines.
+$IsAzurePipelines = "$Env:TF_BUILD" -ne "";
```

Comment:
We normally use the AGENT_ID variable for this.

Reply:
Sure, I can use that one instead